From c9ac2eadfb2c80ddd2cff30c5eaec0f4e8695167 Mon Sep 17 00:00:00 2001
From: David Nadlinger <code@klickverbot.at>
Date: Sat, 19 Feb 2022 20:31:41 +0000
Subject: [PATCH] debuginfo: Fix build on 32-bit ARM

This slipped through in 955d4271, as we aren't building for
32 bit ARM during CI right now.

GitHub: Fixes #44254.
---
 src/debuginfo.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/debuginfo.cpp b/src/debuginfo.cpp
index 42d67bd6f89c..53105feb81bf 100644
--- a/src/debuginfo.cpp
+++ b/src/debuginfo.cpp
@@ -227,7 +227,7 @@ class JITObjectRegistry
                     continue;
                 }
             }
-            uint64_t loadaddr = L.getSectionLoadAddress(section);
+            uint64_t loadaddr = getLoadAddress(section.getName().get());
             size_t seclen = section.getSize();
             if (istext) {
                 arm_text_addr = loadaddr;
